pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -444,21 +440,12 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 			UID: types.UID(volume.podName),
 		},
 	}
-	// TODO: remove feature gate check after no longer needed
-	var mapperPlugin volumepkg.BlockVolumePlugin
-	tmpSpec := &volumepkg.Spec{PersistentVolume: &v1.PersistentVolume{Spec: v1.PersistentVolumeSpec{}}}
-	if utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) {
-		mapperPlugin, err = rc.volumePluginMgr.FindMapperPluginByName(volume.pluginName)
-		if err != nil {
-			return nil, err
-		}
-		tmpSpec = &volumepkg.Spec{PersistentVolume: &v1.PersistentVolume{Spec: v1.PersistentVolumeSpec{VolumeMode: &volume.volumeMode}}}
-	}
-	volumeHandler, err := operationexecutor.NewVolumeHandler(tmpSpec, rc.operationExecutor)
+	mapperPlugin, err := rc.volumePluginMgr.FindMapperPluginByName(volume.pluginName)
```

Comment:
If the feature gate is not enabled, then this will return nil and it's fine?

Reply:
Yes, following sequences doesn't use `mapperPlugin` when feature gate is not enabled.
Just in case, I tested this path again and worked without problem.